tests/test_get_passname

```diff
@@ -25,6 +25,22 @@ test_passname_selection() {
     assert_equals "${items[${select_item}]}" "${output}" "passname: not find valid item"
 }
 
+test_passname_selection_case_insensitive() {
+    local -r select_item=1
+    local -r items=("Item0" "iTeM1" "ITEM2")
+
+    FAKE_PARAMS=(-i)
+
+    fake find fake_find "${HOME}/.password-store" "${items[@]}"
+    fake _wofi fake_wofi ${select_item}
+
+    local output="$(get_passname_from_menu "${FAKE_PARAMS[@]}")"
+
+    assert_equals "${items[${select_item}]}" "${output}" "passname: failed to find item in case-insensitive search"
+
+    FAKE_PARAMS=()
+}
+
```

Comment:
This makes not much sense.
Here we should just check if `--insensitive` args is passed to `wofi` if FLAG_CASE_INSENSITIVE is 1.

Suggestion:
```
is_arg_set() {
    local regex=".*--${2}.*"
    [[ "${1}" =~ ${regex} ]] && return 0 || return 1
}

test_passname_case_insensetive_args() {
    local result
    fake find echo ""
    fake _wofi wofi_args
    local output="$(get_passname_from_menu)"

    result=true
    is_arg_set "${output}" "insensitive" && result=false
    assert ${result} "passname: wofi argument insensitive was unexpected passed"

    FLAG_CASE_INSENSITIVE=1
    output="$(get_passname_from_menu)"
    result=false
    is_arg_set "${output}" "insensitive" && result=true
    assert ${result} "passname: wofi argument insensitive not found"
}
```

**PLEASE TEST THIS EXAMPLE CODE**

Reply:
I did test your example code and they pass. The intention of my test was not to test whether insensitive flag is passed, but whether passing the insensitive flag actually results in finding passwords by not searching for them in a case sensitive way. 

As mentioned earlier, I never wrote unit tests for bash scripts. I found this program, thought it's nice but missing a critical feature, and just wanted to add that.